### PR TITLE
Correct test input file Refs #405

### PR DESCRIPTION
**Pull request by kyriv1980**
This Pr resolves issue #405

### CODE REVIEW

**Comment by GiudGiud** [comment]
This fails even with a new pronghorn it seems
you ll want to update the subchannel module in pronghorn first, then update the pronghorn submodule in the VTB in this PR

**Comment by kyriv1980** [comment]
Ok, i updated the subchannel submodule in the latest pronghorn Pr. Once that gets merged i will update pronghorn in this Pr.

**Comment by GiudGiud** [comment]
you'll need to update the pronghorn submodule. and once it passes tests we can merge

**Comment by GiudGiud** [comment]
looks like we need to investigate a msfr failure

File: msr/generic_msr/seismic_analysis/tests [hunk 1]
```diff
@@ -20,15 +21,18 @@
...
   []
+
+  # Regression tests
   [seismic_analysis1]
     type = CSVDiff
     input = 'basemat_with_isolators_new.i'
     csvdiff = 'basemat_with_isolators_new_out.csv'
     executable_pattern = 'mastodon*'
     abs_zero = 1e-5
     rel_err = 1e-4
```
Comment (GiudGiud) [inline msr/generic_msr/seismic_analysis/tests:32]:
```suggestion
    # Tolerance set from p=4 and p=1 comparison
    rel_err = 4e-4
```

File: msr/generic_msr/seismic_analysis/tests [hunk 2]
```diff
@@ -38,8 +42,9 @@
...
```
Comment (GiudGiud) [inline msr/generic_msr/seismic_analysis/tests:44]:
```suggestion
    # Tolerance set from p=4 and p=1 comparison
    rel_err = 2e-4
```

File: msr/generic_msr/seismic_analysis/tests [hunk 3]
```diff
@@ -37,9 +42,11 @@
     csvdiff = 'basemat_with_isolators_new_3D_out.csv'
     executable_pattern = 'mastodon*'
     abs_zero = 1e-5
-    rel_err = 1e-4
-    max_time = 1000
+    # Tolerance set from p=4 and p=1 comparison
+    rel_err = 2e-4
```
Comment (GiudGiud) [inline msr/generic_msr/seismic_analysis/tests:46]:
```suggestion
    rel_err = 4e-4
```

File: msr/generic_msr/seismic_analysis/tests [hunk 4]
```diff
@@ -20,15 +21,19 @@
...
   [seismic_analysis1]
     type = CSVDiff
     input = 'basemat_with_isolators_new.i'
     csvdiff = 'basemat_with_isolators_new_out.csv'
     executable_pattern = 'mastodon*'
     abs_zero = 1e-5
-    rel_err = 1e-4
-    max_time = 1000
+    # Tolerance set from p=4 and p=1 comparison
+    rel_err = 4e-4
```
Comment (GiudGiud) [inline msr/generic_msr/seismic_analysis/tests:33]:
```suggestion
    rel_err = 2e-3
```

File: msr/generic_msr/seismic_analysis/tests [hunk 5]
```diff
@@ -20,15 +21,19 @@
...
   [seismic_analysis1]
     type = CSVDiff
     input = 'basemat_with_isolators_new.i'
     csvdiff = 'basemat_with_isolators_new_out.csv'
     executable_pattern = 'mastodon*'
     abs_zero = 1e-5
-    rel_err = 1e-4
-    max_time = 1000
+    # Tolerance set from p=4 and p=1 comparison
+    rel_err = 2e-3
```
Comment (GiudGiud) [inline msr/generic_msr/seismic_analysis/tests:33]:
```suggestion
    rel_err = 2.5e-3
```

File: msr/generic_msr/seismic_analysis/tests [hunk 6]
```diff
@@ -37,9 +42,11 @@
     csvdiff = 'basemat_with_isolators_new_3D_out.csv'
     executable_pattern = 'mastodon*'
     abs_zero = 1e-5
-    rel_err = 1e-4
-    max_time = 1000
+    # Tolerance set from p=4 and p=1 comparison
+    rel_err = 4e-4
```
Comment (GiudGiud) [inline msr/generic_msr/seismic_analysis/tests:46]:
```suggestion
    rel_err = 7e-4
```

**Review by GiudGiud — APPROVED** [review]
BC and DW fails are due to those submodules lagging
Cardinal failure unrelated

**Pull request merged**